Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController.swift

```diff
@@ -1817,9 +1817,33 @@ public class BrowserViewController: UIViewController {
         if let url = tab.webView?.url, url.isReaderModeURL {
           break
         }
+        
         tab.secureContentState = .mixedContent
       }
-
+      
+      if let url = tab.webView?.url,
```

Comment:
Hmm this is hasOnlySecureContent right ? which should mean **web view loaded all resources on the page through securely encrypted connections**. Is there a reason we are checking internal is error ?

Reply:
Yes. When you're on a local-host page such as the interstitial pages, it will always show as `hasSecureContentOnly = false` because there's no local ssl cert for those pages.

So when you're offline for example, it will show as insecure even though it's not (no evaluation was done). But it's insecure because `hasSecureContentOnly = false`. So in addition to that check, we have to check if it's our interstitial pages. We know they're secure because they're ours and it's local and it's offline. So we override the check and mark it secure.